Mods/DefensivePositions/Languages/Spanish/DefInjected/KeyBindingCategories.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8" ?>
+
+<KeyBindingCategoryDefs>
```

Comment:
For DefInjected text we don't copy over the whole def, we just specify the text fields to be replaced like so:
`<defName.fieldName>translated text</defName.fieldName>`
Which means, this file should contain instead:
```
<LanguageData>
	<DefensivePositions.description>Accesos directos para las acciones del mod Defensive Positions.</DefensivePositions.description>
</LanguageData>
```